ui/component/channelEdit/view.jsx

```diff
@@ -219,6 +239,10 @@ function ChannelForm(props: Props) {
     clearChannelErrors();
   }, [clearChannelErrors]);
 
+  React.useEffect(() => {
+    claimInitialRewards();
+  }, [claimInitialRewards]);
```

Comment:
Room for optimization to not call this every time?

This will get called every time. Tom did mention something similar:
> _Do this when the balance is less than 1 LBC or something._

@tzarebczan, if there's no harm to the unnecessary API call, I guess we can just merge it.

<img src="https://user-images.githubusercontent.com/64950861/129132705-54006255-e2ef-4687-90ea-72e3bcfe1a5c.png" width="300">

Reply:
Maybe check `selectUnclaimedRewards` first before sending the request?

Reply:
Yea, I thought that was happening..it's probably not a big deal. That's why ```import { selectUnclaimedRewardValue, selectIsClaimingInitialRewards } from 'redux/selectors/rewards';``` was brought in on publish page, but it's never used. @Ruk33 take another look please?

Reply:
Hello guys, thanks for the review. Not sure how much performance can we gain since when we call `doClaimInitialRewards`, internally, it calls `doClaimRewardType` which do checks for already claimed rewards and if it's claimed already, it does nothing. In any case, will update to first make the checks and then call only if required.

Reply:
#### Bug: email verified without user consent
I see what you mean and it is indeed redundant.  On closer inspection, I think the issue is this:
- I don't have rewards approved, but the network call was sent regardless.
- [ ] I got the 0.2 LBC for `Email Verified` despite not having rewards approved (I really didn't want to verify this email, to keep it for testing scenario purposes.).
- [ ] Subsequent visits to "edit" or "publish" results in the screenshot above (repeated network calls).